languages/en.ini

```diff
@@ -277,6 +277,8 @@ confirm_renew_selected_text = "Do you wish to renew the selected items?"
 confirm_storage_retrieval_request_cancel_all_text = "Do you wish to cancel all your current storage retrieval requests?"
 confirm_storage_retrieval_request_cancel_selected_text = "Do you wish to cancel your selected storage retrieval requests?"
 Connecting of library cards is not supported = "Connecting of library cards is not supported"
+consortial_vufind_recommend_heading = "Borrow from a Partner Library"
+consortial_vufind_recommend_intro = '<a href="%%url%%" target="_blank">%%result_count%% result(s)</a> found at partner libraries.'
```

Comment:
It's a project convention to add _html to the end of keys that contain HTML and should not be escaped -- looks like that would be a good idea here.

Reply:
Thanks, I didn't notice that.  Fix.